perfact/zodbsync/tests/test_sync.py

```diff
@@ -869,6 +870,43 @@ def test_watch_structure_changes_and_playback_local_changes(self, conn):
         self.watcher_step_until(watcher,
                                 lambda: "('title', '"+new_title+"')" in meta)
 
+    def test_watch_structure_changes_and_playback_deleted_folder(self, conn):
+        """
+        create structure while 'watch' command is running,
+        remove a folder, then play those changes back and check,
+        whether the step function correctly "crashes"
+        """
+
+        # start watch daemon
+        watcher = self.mkrunner('watch')
+        watcher.setup()
+        app = conn.app
+        folder_1 = "folder_1"
+        s_folder_1 = "s_folder_1"
+
+        # create folder and wait until watch notices change
+        with conn.tm:
+            app.manage_addFolder(id=folder_1)
+        self.watcher_step_until(watcher,
+                                lambda: os.path.isdir(
+                                    self.repo.path + '/__root__/'+folder_1))
+
+        # create subfolder and wait until watch notices change
+        with conn.tm:
+            app.folder_1.manage_addFolder(id=s_folder_1, title=s_folder_1)
+        path = self.repo.path + '/__root__/'+folder_1+'/'+s_folder_1
```

Comment:
Interesting that `flake8` does not complain about missing spaces around the operators here.